src/api.js

```diff
@@ -19,6 +19,22 @@ export const incVote = (reviewID, increment) => {
 };
 
 export const fetchComments = (reviewID) => {
-  console.log(reviewID);
   return api.get(`reviews/${reviewID}/comments`).then((res) => res.data);
 };
+
+export const postComment = (reviewID, newComment, currentUser) => {
+  return api
+    .post(`reviews/${reviewID}/comments`, {
+      username: currentUser,
+      body: newComment,
+    })
+    .then((res) => res.data);
+};
+
+export const fetchCategoryBySlug = (category_slug) => {
+  return api.get(`reviews?category=${category_slug}`).then((res) => res.data);
+};
```

Comment:
Could you refactor your code to re-use the <Reviews /> component and the fetchReviews api request?
Look at axios query object on how to send queries i.e. ?category=${category_slug}`

https://masteringjs.io/tutorials/axios/get-query-params